py-rse/bash-basics.Rmd

```diff
@@ -1689,24 +1790,22 @@ $ history
 ```
 
 ```text
-  ...
-  571  wc -l *.csv | sort -n | head n 3
-  572  wc -l *.csv | sort -n | head -n 3
-  573  wc -l *.csv | sort -n | head -n 3 | tail -n 1
-  ...
+  551  wc -l *.txt | sort -n
+  552  wc -l *.txt | sort -n | head -n 3
+  553  wc -l *.txt | sort -n | head -n 3 | tail -n 1
```

Comment:
If you agree with my earlier comment regarding running `wc -l *.txt | sort -n | head -n 1` instead of command 552 and 553 above, then this output from `history` will need to be updated.